frontend/public/components/namespace.jsx

```diff
@@ -270,6 +270,7 @@ class NamespaceDropdown_ extends React.Component {
         className="co-namespace-selector__dropdown"
         menuClassName="co-namespace-selector__menu"
         noButton
+        namespace
```

Comment:
@rebeccaalpert I'm still not clear why we're special casing the namespace dropdown. The dropdown component shouldn't need to know anything about the namespace picker.